### DISCUSSION TITLE
Wide Textfield

### DISCUSSION THREAD

**Discussion by Corrsy0** [flet-dev/flet#2345] [Q&A]
I tried to create a wide textfield but there is no such feature. I tried something like this but I don't want to do it like this.

ft.TextField(
    bgcolor="white",
    color="gray",
    label="Linkler" if page.config["lang"] == "turkey" else "Links",
    text_size=16,
    width=700,
    height=150,
    multiline=True,
    value="\n\n\n\n", # The method I tried to expand the textfield
    on_change=change
)

**Comment by Corrsy0** [comment]
bumped

**Comment by ingdesarr1** [comment]
you need to add this properties in the TextField. if put min in a value >1, the Textfield show wide.

`
            min_lines=1,
            max_lines=3,
`
https://flet.dev/docs/controls/textfield#min_lines
